app/Http/Controllers/Admin/SettingsController.php

```diff
@@ -111,9 +123,22 @@ public function setCSS(SetCSSSettingRequest $request): void
 	{
 		$css = $request->getCss();
 		if (Storage::disk('dist')->put('user.css', $css) === false) {
+			// @codeCoverageIgnoreStart
+			// We do not test this part as this would require to change the access rights of the file
 			if (Storage::disk('dist')->get('user.css') !== $css) {
 				throw new InsufficientFilesystemPermissions('Could not save CSS');
 			}
+			// @codeCoverageIgnoreEnd
 		}
 	}
-}
\ No newline at end of file
+
+	/**
+	 * Return the necessary information to configure the settings page.
+	 *
+	 * @return SettingsConfig
+	 */
+	public function getConfig(GetAllConfigsRequest $request): SettingsConfig
```

Comment:
I think we don't need the `$request` parameter? beacuse we are returning new Settings config all the time?